### PR TITLE
Building management

### PR DESCRIPTION
**Pull request by huang831206**

### CODE REVIEW

File: app/Http/Controllers/BuildingController.php [hunk 1]
```diff
@@ -87,7 +150,55 @@ public function edit(Building $building)
...
+            'administrative_number' => 'required|max:255',
+            'accounting_group' => 'required|max:255',
+            'rental_receipt' => 'required|max:255',
+            'commissioner_id' => 'nullable|exists:users,id',
+            'administrator_id' => 'nullable|exists:users,id',
+            'comment' => 'required|max:255',
+        ]);
+
+        $landlord->update($validatedData);
+
```
Comment (nick66551) [inline app/Http/Controllers/BuildingController.php]: `$landlord` ?

File: resources/views/buildings/table.blade.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            @if($model_name == null)
+               {{$layer}}
+            @else
+                @lang("model.{$model_name}.{$layer}")
+            @endif
+        </h2>
+
+        {{-- the route to create this kind of resource --}}
+        <a class="btn btn-sm btn-success" href="{{ route( 'buildings.create') }}">建立</a>
+
```
Comment (nick66551) [inline resources/views/buildings/table.blade.php]: Help me adjust to :
`<a class="btn btn-sm btn-success my-3" href="{{ route( 'buildings.create') }}">建立</a>`
Thanks

File: resources/views/rooms/table.blade.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            @if($model_name == null)
+               {{$layer}}
+            @else
+                @lang("model.{$model_name}.{$layer}")
+            @endif
+        </h2>
+
+        {{-- the route to create this kind of resource --}}
+        <a class="btn btn-sm btn-success" href="{{ route( 'rooms.create') }}">建立</a>
+
```
Comment (nick66551) [inline resources/views/rooms/table.blade.php]: Help me adjust to :
`<a class="btn btn-sm btn-success my-3" href="{{ route( 'rooms.create') }}">建立</a>`
Thanks

**Comment by huang831206** [comment]
room curd added

**Pull request merged**